src/stores/language-store.ts

```diff
@@ -9,7 +9,9 @@ export class LanguageStore {
   public language: string;
 
   constructor(private rootStore: RootStore) {
-    this.language = RNLocalize.findBestAvailableLanguage(languages).languageTag;
+    this.language = RNLocalize.findBestAvailableLanguage(
+      languages.map((l) => l.key)
+    ).languageTag;
```

Comment:
does it need failsafe?

Reply:
No it doesn't. It will find a language, everytime